src/odemis/gui/cont/tabs.py

```diff
@@ -3278,8 +3244,8 @@ def __init__(self, name, button, panel, main_frame, main_data):
         else:
             self.panel.btn_manualfocus.Show(True)  # TODO only for streak cam now
             # Bind event
-            self.panel.btn_open_streak_calib_file.Bind(wx.EVT_BUTTON, self._onOpenCalibFile)
-            self.panel.btn_save_streak_calib_file.Bind(wx.EVT_BUTTON, self._onSaveCalibFile)
+            self.panel.btn_open_streak_calib_file.Bind(wx.EVT_BUTTON, self._settings_controller._onOpenCalibFile)
```

Comment:
The `StreakCamAlignSettingsController` should be able to bind it itself. It may look like the `if` condition is different, but actually it's not, as `streak-align` be added to the alignment modes based on the hardware present.... and anyway, if the button is shown, it should be bound to a function.